stacker_blueprints/rds/aurora/base.py

```diff
@@ -179,6 +184,18 @@ def create_security_group(self):
                 )
             )
             self.security_group = Ref(sg)
+
+            if variables["AllowedCIDRs"]:
```

Comment:
Actually - there's an issue with that, in that the rules wouldn't have a `SecurityGroup` attribute, which is required for the `SecurityGroupIngress` type.  That would normally be set to the SG that is created here.  Though didn't you just add a feature to `TroposphereType` that allows for modifying the resource after the fact, not requiring immediate validation? If so, then that'd work with this.  Otherwise you could do what we did in the `security_rules.py` blueprint and just accept a dict, then use `from_dict` after you've added the `SecurityGroup` attribute.